packages/api/tests/endpoints/collections.js

```diff
@@ -0,0 +1,285 @@
+'use strict';
+
+const sinon = require('sinon');
+const test = require('ava');
+const aws = require('@cumulus/common/aws');
+const { randomString } = require('@cumulus/common/test-utils');
+const models = require('../../models');
+const bootstrap = require('../../lambdas/bootstrap');
+const collectionsEndpoint = require('../../endpoints/collections');
+const EsCollection = require('../../es/collections');
+const {
+  fakeCollectionFactory,
+  fakeUserFactory,
+  testEndpoint
+} = require('../../lib/testUtils');
+const { Search } = require('../../es/search');
+const assertions = require('../../lib/assertions');
+
+process.env.CollectionsTable = randomString();
+process.env.UsersTable = randomString();
+process.env.stackName = randomString();
+process.env.internal = randomString();
+
+const testCollection = fakeCollectionFactory();
+const esIndex = randomString();
+
+let authHeaders;
+let collectionModel;
+let userModel;
+test.before(async () => {
+  await bootstrap.bootstrapElasticSearch('fakehost', esIndex);
+  sinon.stub(EsCollection.prototype, 'getStats').returns([testCollection]);
+  await aws.s3().createBucket({ Bucket: process.env.internal }).promise();
+
+  collectionModel = new models.Collection({ tableName: process.env.CollectionsTable });
+  await collectionModel.createTable();
+  await collectionModel.create(testCollection);
+
+  // create fake Users table
+  userModel = new models.User();
+  await userModel.createTable();
+
+  const authToken = (await userModel.create(fakeUserFactory())).password;
+  authHeaders = {
+    Authorization: `Bearer ${authToken}`
+  };
+});
+
+test.after.always(async () => {
+  await collectionModel.deleteTable();
+  await userModel.deleteTable();
+  await aws.recursivelyDeleteS3Bucket(process.env.internal);
+
+  const esClient = await Search.es('fakehost');
+  await esClient.indices.delete({ index: esIndex });
+});
+
+test('CUMULUS-911 GET without pathParameters and without an Authorization header returns an Authorization Missing response', async (t) => {
```

Comment:
Is there a reason we have the ticket number in these? Is it just for the review?

Reply:
It seemed like a good practice to get into so that, if someone ever looks at the test and wants to know more about why we're testing that behavior, they can.  It should also make it easier to map back to acceptance criteria.